contentpacks/update_dubbed_video_spreadsheet.py

```diff
@@ -0,0 +1,269 @@
+    language name.
+
+# Reference
+    * https://github.com/burnash/gspread
+    * https://github.com/burnash/gspread/issues/201
+"""
+
+import errno
+import gspread
+import os
+import logging
+import requests
+import string
+import sys
+import ujson
+
+from contentpacks.utils import NodeType
+from oauth2client.service_account import ServiceAccountCredentials
+
+
+PROJECT_PATH = os.path.join(os.getcwd())
+BUILD_PATH = os.path.join(PROJECT_PATH, "build")
+GOOGLE_CREDENTIAL_PATH = os.path.join(BUILD_PATH, "credential", 'credentials.json')
+EN_LANGUAGELOOKUP = "english"
+EN_LANG_CODE = "en"
+BUILD_VERSION = "0.17.x"
+SPREADSHEET_DEFAULT_VALUE = [
+    "SERIAL", "DATE ADDED", "DATE CREATED", "TITLE", "LICENSE", "DOMAIN", "SUBJECT", "TOPIC", "TUTORIAL", "TITLE ID",
+    "URL", "DURATION", "REQUIRED FOR", "TRANSCRIPT"]
+
+LE_SUPPORTED_LANG = ['arabic', 'armenian', 'bahasa indonesia', 'bangla', 'bulgarian', 'chinese', 'czech',
+                     'danish', 'dari', 'deutsch', 'english', 'espanol', 'farsi', 'francais', 'greek', 'hebrew',
+                     'hindi', 'italiano', 'japanese', 'kiswahili', 'korean', 'mongolian', 'nederlands', 'norsk',
+                     'polish', 'portugal portugues', 'portugues', 'punjabi', 'russian', 'serbian', 'sindhi',
+                     'sinhala', 'tamil', 'telugu', 'thai', 'turkce', 'ukrainian', 'urdu', 'xhosa', 'zulu']
+
+logging.getLogger().setLevel(logging.INFO)
+
+def _ensure_dir(path):
+    """Create the entire directory path, if it doesn't exist already."""
+    try:
+        os.makedirs(path)
+    except OSError as e:
+        if e.errno == errno.EEXIST:
+            # file already exists
+            if not os.path.isdir(path):
+                # file exists but is not a directory
+                raise OSError(errno.ENOTDIR, "Not a directory: '%s'" % path)
+            pass  # directory already exists
+        else:
+            raise
+        
+
+def convert_to_json(lang_url, lang_code):
+    data = requests.get(lang_url)
+    node_data = ujson.loads(data.content)
+    dump_json = os.path.join(BUILD_PATH, "%s_node_data.json" % lang_code)
+    with open(dump_json, "w") as f:
+        ujson.dump(node_data, f)
+        
+
+def _access_google_spreadsheet():
+    credentials = None
+    _ensure_dir(os.path.dirname(GOOGLE_CREDENTIAL_PATH))
+    scope = ['https://spreadsheets.google.com/feeds']
+    if os.path.exists(GOOGLE_CREDENTIAL_PATH):
+        credentials = ServiceAccountCredentials.from_json_keyfile_name(GOOGLE_CREDENTIAL_PATH, scope)
+    else:
+        logging.info("Please create your google credentials.")
+    gcreadentials = gspread.authorize(credentials)
+    sheet = gcreadentials.open_by_url("https://docs.google.com/spreadsheets/d/1q9fVt5cxkR7dI7uLR1klGDK7XHjjeSvBoWYilkDqE50/edit#gid=0")
+    return sheet
+
+
+def get_en_data():
+    BUILD_PATH = "/Users/mrpau-eduard/content-pack-maker/build"
```

Comment:
There are also two other issues:

1. Writing in CAPITALS suggest that it's a global variable, however it's not declared as such.
1. Should always use `os.path.join` for paths, even though the code isn't expected to run on Windows... make it a habbit, it will make you happier in the future :)